src/widget/wlibrarytableview.cpp

```diff
@@ -112,6 +111,21 @@ void WLibraryTableView::moveSelection(int delta) {
     }
 }
 
+void WLibraryTableView::saveVScrollBarPos(const QString key){
+    m_vScrollBarPosValues[key] = verticalScrollBar()->value();
+}
+
+void WLibraryTableView::restoreVScrollBarPos(const QString key){
+    updateGeometries();
+
+    if (m_vScrollBarPosValues.contains(key)){
+        verticalScrollBar()->setValue(m_vScrollBarPosValues[key]);
```

Comment:
this is a double map lookup. use find() and check for end() like here: 
https://github.com/mixxxdj/mixxx/blob/a944d1f224788e8efb41ad41bd487210766ddb11/src/control/control.cpp#L88